ClientApp/src/components/Admin/AdminInterface.tsx

```diff
@@ -19,18 +21,22 @@ const AdminInterface = (): JSX.Element => {
       null,
       'ADMIN_SETTINGS_NOT_FOUND',
       (responseJSON: FixTypeLater[]): void => {
-        setAdminSettings(responseJSON.map(s => plainToClass(AdminSetting, s)))
+        setAdminSettings(
+          responseJSON.map((s) => plainToInstance(AdminSetting, s))
+        )
       }
     )
   }, [])
 
+  console.log('adminSettings', adminSettings)
```

Comment:
What are contained in the adminSettings? Does this need to be printed to the log? I imagine it's there for debugging purposes, but unsure if it should be or not.

Reply:
Will remove.